manifests/vizier/db/deployment.yaml

```diff
@@ -28,10 +28,10 @@ spec:
         ports:
         - name: dbapi
           containerPort: 3306
-#        resources:
-#          requests:
-#            cpu: 500m
-#            memory: 500M
-#          limits:
-#            cpu: 500m
-#            memory: 500M
+        volumeMounts:
+        - name: katib-volume
```

Comment:
I think better name would be katib-mysql or katib-db. We might need more volumes later